src/pages/Authentication/SignUp/SignUp.jsx

```diff
@@ -1,18 +1,56 @@
-import { Link } from "react-router-dom";
+import { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
```

Comment:
Can have better components name inputtype one two three is not much easily understandable

Reply:
Will look into it.